homeassistant/const.py

```diff
@@ -228,6 +228,10 @@
 VOLUME_GALLONS = 'gal'  # type: str
 VOLUME_FLUID_OUNCE = 'fl. oz.'  # type: str
 
+# Bandwidth units
+BANDWIDTH_MEGABITS_SECONDS = 'Mbps'  # type: str
+BANDWIDTH_GIGABITS_SECONDS = 'Gbps'  # type: str
```

Comment:
Ok maybe for this PR I will put these units in the sensor file.  I will look for other composent that used this kind of units. If there are a few component usine thèses, I will make another PR to gather all in the const.py file